src/domain/event/__tests__/EditEventPage.test.tsx

```diff
@@ -244,52 +244,50 @@ describe('Language selection', () => {
 
   test('translatable fields appear in selected languages', async () => {
     const { user } = renderComponent();
+    await actWait();
     const languageSelector = await screen.findByTestId(
       formLanguageSelectorTestId
     );
-
     // Select Swedish (with Finnish that is already selected)
     const sv = await within(languageSelector).findByRole('checkbox', {
       name: /ruotsi/i,
     });
     await user.click(sv);
+    await actWait();
```

Comment:
Should be useless here. How about using this once in the end of the test?

https://kentcdodds.com/blog/fix-the-not-wrapped-in-act-warning

Reply:
Removed most uses of `await actWait();` in 5fe6477e98db1ed3a2d774936736a99dc66aa63f